csharp/language-reference/builtin-types/UnmanagedTypes.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+public struct Coords<T>
+{
+    public T X;
+    public T Y;
+}
+
+public class UnmanagedTypes
```

Comment:
I didn't test in an IDE, but I think this class can be made static.

Reply:
It can be static, but I don't think it's necessary.

Reply:
In other similar occasions where the full program is in snippet, it's not static. I'd follow the same pattern here.